### PR TITLE
docs(plugins): update plugin listing and add Asana instructions

### PR DESCRIPTION
**Pull request by scefali**
With plugin repos being archived, that information needed to be propagated here Our Asana plugin has instructions for on-prem users so they should be added here. I did not add instructions the existing instructions for BitBucket and Jira because those are deprecated and have replacements.

### CODE REVIEW

File: src/collections/_documentation/server/plugins.md [hunk 1]
```diff
@@ -47,41 +47,41 @@ All official plugins are tested against the latest version of Sentry, and compat
...
-* Bitbucket
-* Visual Studio Team Services
 * Jira
+* Slack
+* Visual Studio Team Services
 
 ## 3rd Party Plugins {#rd-party-plugins}
 
-The following plugins are available and maintained by members of the Sentry community:
+The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained may not work with the latest versions of sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/server/plugins.md]:
```suggestion
The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained may not work with the latest versions of Sentry.
```

File: src/collections/_documentation/server/plugins.md [hunk 2]
```diff
@@ -100,3 +100,18 @@ The following plugins are available and maintained by members of the Sentry comm
 -   [sentry-youtrack](https://github.com/bogdal/sentry-youtrack)
 -   [sentry-zabbix](https://github.com/m0n5t3r/sentry-zabbix)
 -   [sentry-zendesk](https://github.com/ESSS/sentry-zendesk)
+
+## Setup
+
+Many of these plugins require configuration to work. The instruction on how to configure your environment for these plugins are contianed below.
```
Comment (MimiDumpling) [inline src/collections/_documentation/server/plugins.md]:
```suggestion
Many of these plugins require configuration to work. The instructions on how to configure your environment for these plugins are below.
```

File: src/collections/_documentation/server/plugins.md [hunk 3]
```diff
@@ -100,3 +100,18 @@ The following plugins are available and maintained by members of the Sentry comm
...
+
+### Asana
+
+You'll have to create an application in Asana to get a client ID and secret. Use the following for the redirect URL:
+
+    <URL_TO_SENTRY>/account/settings/social/associate/complete/asana/
+
+Ensure you've configured Asana auth in Sentry::
```
Comment (MimiDumpling) [inline src/collections/_documentation/server/plugins.md]:
```suggestion
Ensure you've configured Asana auth in Sentry:
```

**Comment by scefali** [comment]
@MimiDumpling anything else?

File: src/collections/_documentation/server/plugins.md [hunk 4]
```diff
@@ -47,41 +47,41 @@ All official plugins are tested against the latest version of Sentry, and compat
...
-* Bitbucket
-* Visual Studio Team Services
 * Jira
+* Slack
+* Visual Studio Team Services
 
 ## 3rd Party Plugins {#rd-party-plugins}
 
-The following plugins are available and maintained by members of the Sentry community:
+The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained may not work with the latest versions of Sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/server/plugins.md]:
```suggestion
The following plugins are available and were created by members of the Sentry community. Please note that many of these plugins have not been maintained and may not work with the latest versions of Sentry.
```

**Pull request merged**